rotkehlchen/chain/aggregator.py

```diff
@@ -342,6 +342,14 @@ def deactivate_premium_status(self) -> None:
             if hasattr(module, 'premium') is True and module.premium is not None:  # type: ignore
                 module.premium = None  # type: ignore
 
+        # Also flush the cache of anything that is touched by eth2 validators since
+        # without premium we have a limit
+        self.flush_cache('query_eth2_balances')
+        self.flush_cache('query_balances')
+        self.flush_cache('query_balances', blockchain=SupportedBlockchain.ETHEREUM_BEACONCHAIN)
+        self.flush_cache('query_balances', blockchain=None, ignore_cache=False)
+        self.flush_cache('query_balances', blockchain=SupportedBlockchain.ETHEREUM_BEACONCHAIN, ignore_cache=False)  # noqa: E501
```

Comment:
is it needed to clear that last one? ignore cache should ignore the cache 😓

Reply:
we already do this in other places we clean the cache. You basically clean all possible argument combinations. It's dirty I know.